.editorconfig

```diff
@@ -63,7 +63,7 @@ csharp_style_throw_expression = true:suggestion
 csharp_style_conditional_delegate_call = true:suggestion
 
 # Expression-level preferences
-csharp_prefer_braces = false:none
+csharp_prefer_braces = false:suggestion
```

Comment:
@jcansdale Did you mean to make this `true:suggestion`?

Reply:
I did, thanks for the catch!